### PR TITLE
add 1.8 and lower version timespine

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds [older content](https://github.com/dbt-labs/docs.getdbt.com/blob/44d405fdc0580554fa15ba721b709ee2537e74aa/website/docs/docs/build/metricflow-time-spine.md?plain=1) for 1.8 and lower to timespine docs in order to avoid user confusion.

[slack thread here](https://getdbt.slack.com/archives/C046L0VTVR6/p1728318386744829)

### CODE REVIEW

File: website/docs/docs/build/metricflow-time-spine.md [hunk 1]
```diff
@@ -290,6 +294,239 @@ and date_hour < dateadd(day, 30, current_timestamp())
 </File>
 
 
+</VersionBlock>
+
+<VersionBlock lastVersion="1.8">
+
+<!-- this whole section is for 1.8 and and lower -->
```
Comment (mirnawong1) [inline website/docs/docs/build/metricflow-time-spine.md:301]: this is the v1.8 and lower section

File: website/docs/docs/build/metricflow-time-spine.md [hunk 2]
```diff
@@ -290,6 +294,239 @@ and date_hour < dateadd(day, 30, current_timestamp())
...
+{{
+    config(
+        materialized = 'table',
+    )
+}}
+
+with hours as (
+
+    {{
+        dbt.date_spine(
```
Comment (mirnawong1) [inline website/docs/docs/build/metricflow-time-spine.md]: should this be dbt_utils.date_spine for 1.6 and lower? and dbt.date_spine for 1.7 and higher?

Comment (courtneyholcomb) [reply website/docs/docs/build/metricflow-time-spine.md]: Oooo I'm actually not sure on this one!

File: website/docs/docs/build/metricflow-time-spine.md [hunk 3]
```diff
@@ -74,6 +76,7 @@ This example creates a time spine at an hourly grain and a daily grain: `time_sp
 
 <Lightbox src="/img/time_spines.png" width="50%" title="Time spine directory structure" />
 
+<!--
```
Comment (courtneyholcomb) [inline website/docs/docs/build/metricflow-time-spine.md:79]: Is this commenting out the version block below?

Comment (mirnawong1) [reply website/docs/docs/build/metricflow-time-spine.md:79]: yes it is! but we dont need this yaml for 1.8 and lower right?

**Review by courtneyholcomb — APPROVED** [review]

**Pull request merged**